media/js/firefox/family-index.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+;(function($, _gaq, site) {
+    'use strict';
+
+    var $html = $(document.documentElement);
+    var $downloadbar = $('#conditional-download-bar');
+
+    // Open external links in new window/tab
+    $('.product a[rel="external"]').attr('target', '_blank');
+
+    // Dismiss the conditional download bar
+    $downloadbar.on('click', '.btn-close', function() {
+        $downloadbar.animate({
+            top: '-' + ($downloadbar.height() + 10)
+        }, 350, function() {
+            $downloadbar.fadeOut().remove();
+        });
+    });
```

Comment:
Fade out is a bit choppy. Adding `position: relative;` makes the `top` value animate. Moving `remove()` to a callback in `fadeOut` helps, too. Something like:
```
$downloadbar.fadeOut('fast', function() {
    $downloadbar.remove();
});
```